workshops/rock_paper_scissors/README.md

```diff
@@ -0,0 +1,274 @@
+
+### 2)Giving values to the variables
+First let's make a list for all th entries a plyer could enter,
+`t = ["r","p","s"]`
+
+Now we have to write a command so the computer chooses rock, paper or scissors randomly!
+`computer = t[randint(0,2)]`.
+
+Now we will set the player to false so we can keep the game in a loop to do that just type, `Player = False`. Now you have to write some code so the player could enter his/her name. 
+Also to make a score system enter the following code:
+```
+You = 0
+PC = 0 
+```
+So this is the code so far:
+```
+from random import randint
+
+t = ["r", "p", "s"]
+
+computer = t[randint(0,2)]
+
+player = False
+
+name = input("Enter your name:")
+print("Type 'reset' to reset score")
+
+You = 0
+PC = 0
+```
+
+### 3)Keeping the game in a loop
+![Image](https://cloud-nmra250be.vercel.app/loop.gif)
+
+For this game I will using `while` loop but you can also use `for` loop! 
+
+So for starters type,
+```
+While Player == False: 
+```
+Now we let the player choose either `Rock` `Paper` or `Scissors`
+
+There are 5 possibilities:
+1. The player chooses `Rock`
+2. The player chooses `Paper`
+3. The player chooses `Scissors`
+4. The word entered is invalid
+5. The player enters `reset`
+
+### 4) Producing separate outcomes for all possibilities
+
+We are going to use if elif & else statement to produce separate outcomes for all possibilities!
+
+Enter the following code:
+```
+while player == False:
+    player = input("Rock, Paper, Scissors?(r,p,s)")
+    if player == computer:
+        print("Tie!")
+        print('Computer =',PC)
+        print(name,'=',You)
+    elif player == "r":
+        if computer == "p":
+            print("You lose!","Paper covers Rock")
+            PC+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+        else:
+            print("You win!","Rock smashes Scissors")
+            You+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+    elif player == "p":
+        if computer == "s":
+            print("You lose!", "Scissors cut Paper")
+            PC+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+        else:
+            print("You win!","Paper covers Rock")
+            You+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+    elif player == "s":
+        if computer == "r":
+            print("You lose...","Rock smashes Scissors")
+            PC+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+        else:
+            print("You win!", "Scissors cuts Paper")
+            You+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+    elif player == "reset":
+      You=1*0
+      PC=1*0
+      print("The score has been reset!")
+    else:
+        print("That's not a valid play. Please select a valid option!")
+```
+#### Explanation:
+
+1. The first if statement declares a tie.
```

Comment:
Also, just like above, please make it more elaborate.

Reply:
Most of the code is repetition so explaining it separately will just make the workshop wordy